entries/QUnit.push.xml

```diff
@@ -1,32 +0,0 @@
-<?xml version="1.0"?>
-<?xml-stylesheet type="text/xsl" href="../entries2html.xsl" ?>
-<entry type="method" name="QUnit.push">
-	<title>QUnit.push()</title>
-	<signature>
-		<argument name="result" type="Boolean">
-			<desc>Result of the assertion</desc>
-		</argument>
-		<argument name="actual" type="Object">
-			<desc>Expression being tested</desc>
-		</argument>
-		<argument name="expected" type="Object">
-			<desc>Known comparison value</desc>
-		</argument>
-		<argument name="message" type="String" optional="true">
-			<desc>A short description of the assertion</desc>
-		</argument>
-	</signature>
-	<desc>DEPRECATED: Report the result of a custom assertion</desc>
-	<longdesc>
-		<p class="warning">
-			This method is <strong>deprecated</strong> and it's recommended to use it through its direct reference in the assertion context.
-		</p>
-		<p>
-			<code>QUnit.push</code> reflects to the current running test, and it may leak assertions in asynchronous mode. Checkout <a href="/pushResult/">assert.pushResult()</a> to set a proper custom assertion.
-		</p>
-		<p>
-			Invoking <code>QUnit.push</code> allows to create a readable expectation that is not defined by any of QUnit's built-in assertions.
-		</p>
-	</longdesc>
-	<category slug="config"/>
-</entry>
```

Comment:
this file needs to be maintained.

The deprecation message must be updated to reflect this method won't change until the next major version, not 2.0.